service/controller/chart/v1/resource_set.go

```diff
@@ -93,10 +94,23 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		}
 	}
 
+	var tillerResource controller.Resource
+	{
+		c := tiller.Config{
+			HelmClient: config.HelmClient,
+			Logger:     config.Logger,
+		}
+
+		tillerResource, err = tiller.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
 	resources := []controller.Resource{
-		// statusResource executed first so the status is set for existing CRs.
-		statusResource,
+		tillerResource,
 		releaseResource,
+		statusResource,
```

Comment:
@xh3b4sd Note I moved the status resource to last.

I had problems when testing locally. It also means the status resource reflects any changes made by the release resource.